src/test/java/org/mastodon/mamut/collaboration/utils/ModelAsserts.java

```diff
@@ -0,0 +1,183 @@
+package org.mastodon.mamut.collaboration.utils;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.mastodon.mamut.model.Link;
+import org.mastodon.mamut.model.Model;
+import org.mastodon.mamut.model.ModelGraph;
+import org.mastodon.mamut.model.Spot;
+import org.mastodon.model.tag.ObjTagMap;
+import org.mastodon.model.tag.TagSetModel;
+import org.mastodon.model.tag.TagSetStructure;
+
+/**
+ * Utility class that provides static methods to assert that two Mastodon
+ * {@link Model}s are equal.
+ */
+public class ModelAsserts
+{
+	public static void assertModelEquals( Model a, Model b )
```

Comment:
A javadoc would help here to understand more quickly, what is covered by this and what not, e.g. spot coordinates/covariances/names yes/no